i18n/en.json

```diff
@@ -466,6 +466,12 @@
   "admin.experimental.emailBatchingInterval.desc": "Specify the maximum frequency, in seconds, which the batching job checks for new notifications. Longer batching intervals will increase performance.",
   "admin.experimental.emailBatchingInterval.example": "E.g.: \"30\"",
   "admin.experimental.emailBatchingInterval.title": "Email Batching Interval",
+  "admin.experimental.emailSettingsLoginButtonBorderColor.desc": "Specify the color of the Email login button border for white labeling purposes. Use a hex code with a #-sign before the code. This setting only applies to the mobile apps.",
```

Comment:
For each description, I'd lean towards `Email` < `email` because we seem to use lower capitals elsewhere in System Console